Event/Event_Heal.gd

```diff
@@ -0,0 +1,8 @@
+extends Event
+class_name Event_Heal
```

Comment:
rename to EventHeal. I need to modify the contributing to include the class name conventions...